wasp-ai/src/server/operations.ts

```diff
@@ -222,7 +209,46 @@ export const getStats = (async (_args, context) => {
 }) satisfies GetStats<{}>;
 
 export const getNumProjects = (async (_args, context) => {
-  const { Project } = context.entities;
-  const numProjects = await Project.count();
-  return numProjects;
+  return context.entities.Project.count();
 }) satisfies GetNumProjects<{}>;
+
+export const getProjectsByUser: GetProjectsByUser<void, Project[]> = async (_args, context) => {
+  if (!context.user) {
+    throw new HttpError(401, "Not authorized");
+  }
+  return await context.entities.Project.findMany({
+    where: {
+      user: {
+        id: context.user.id,
+      },
+    },
+  });
+};
+
+import { CheckIfUserStarredWasp } from "@wasp/queries/types";
```

Comment:
Ok wild, import in the middle of the document :D. Maybe we should put it at the top of the document?